recipes/stir/meta.yaml

```diff
@@ -0,0 +1,52 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Is there a reason to pull from your personal fork rather than the UCL repo?

Reply:
Yes, the UCL repo has only old archives. I wanted to install a recent version, so I made an archive on my branch.

Reply:
I'd much rather tag a specific commit from the UCL repo, e.g. `https://github.com/UCL/STIR/archive/5710d438090a972bd7b0636ff6de5315519b78e9.tar.gz`, than go off a fork just because you gave it a git tag.

`v3_0_1` is also not an acceptable version number.

- First, the `version` would need to be `3.0.1`; you could use Jinja to make that happen, e.g. `{% set version = "3.0.1" %}` and then in the URL use something like `.../archive/v{{ version | replace(".", "_") }}`.

- Second, we can't just randomly call the current state of `master` `3.0.1` because you want to include some stuff that merged in after `3.0.0`. What happens when the project releases a real `3.0.1`? You _could_ call it `3.0.1.dev1`, but it'd be much nicer to just ask the project to create a current release.

Reply:
Very good point. Hopefully there will soon be a release at the UCL/STIR repo.